webapp/imports/api/methods.js

```diff
@@ -39,14 +38,17 @@ Meteor.methods({
   'games.join'(gameId) {
     check(gameId, String)
 
-    ensureLoggedIn.bind(this)
+    let user = Meteor.user()
+
+    ensureLoggedIn(this.userId)
```

Comment:
I'd say move this before `let user = Meteor.user()` to be really clear you're not messing around with security.